media/css/mozorg/webvision.scss

```diff
@@ -70,37 +70,55 @@ $image-path: '/media/protocol/img';
 
     @media #{$mq-md} {
         flex-direction: row;
-
-        .c-webvision-article {
-            margin-left: $layout-sm;
-        }
     }
 
-    .webvision-sidebar {
-        position: sticky;
-        max-height: 100vh;
-        top: 0;
-        overflow-y: auto;
-        overflow-x: hidden;
-    }
-
-    .mzp-c-sidemenu-main {
+    .c-webvision-article {
 
         @media #{$mq-md} {
-            padding: $layout-lg 0;
+            margin-right: 0;
         }
 
-        @media #{$mq-xl} {
-            width: 350px;
+        @media #{$mq-lg} {
+            margin-right: $layout-2xl;
         }
     }
+}
+
+.webvision-sidebar {
+    position: sticky;
+    max-height: 100vh;
+    top: 80px;
+    overflow-y: auto;
+    overflow-x: hidden;
+}
+
+.mzp-c-sidemenu-main {
+
+    @media #{$mq-md} {
+        padding-bottom: $layout-xl;
+    }
+
+    @media #{$mq-xl} {
+        width: 350px;
+    }
+}
+
+.toc {
+    padding: $spacing-xs;
 
-    .toc {
-        padding: $spacing-xs;
+    ul {
+        li {
+            margin: $spacing-sm;
+            font-weight: 600;
 
-        ul {
-            li {
-                margin: $spacing-sm;
+            a {
+                padding: 0;
```

Comment:
Remove this and the base component styles will take over so we get the original spacing back.

Reply:
But I thought we were reducing the original spacing to make the sidemenu smaller in length?

Reply:
It's still reduced by changing the margin above. The original component had spacing, this page was adding to that by adding more margins. I think we want the original spacing without the extra margins.

Protocol component:
![image](https://user-images.githubusercontent.com/205591/163880203-86c8c157-c17d-4318-8a0c-c91aa44a8413.png)

Extra space added:
![image](https://user-images.githubusercontent.com/205591/163880098-be7a8d3a-0971-41f6-a657-f558e57334f0.png)